src/components/metadataEditor/metadataEditor.js

```diff
@@ -541,7 +541,7 @@ function setFieldVisibilities(context, item) {
         hideElement('#fldPath', context);
     }
 
-    if (item.Type === 'Series' || item.Type === 'Movie' || item.Type === 'Trailer' || item.Type === 'Person') {
+    if (item.Type === 'Series' || item.Type === 'Season' || item.Type === 'Episode' || item.Type === 'Movie' || item.Type === 'Trailer' || item.Type === 'Person') {
```

Comment:
We should do something like:

```js
    if ([ 'Series', 'Season', 'Episode', ... ].includes(item.Type)) {
```

Bonus points if you also use the `BaseItemKind` enum values instead of hardcoded strings. :wink: